src/shared-components/SortingButton/SortingButton.spec.tsx

```
@@ -13,36 +12,50 @@ beforeEach(() => {
+          >
+            {icon}
+            {buttonLabel}
+          </button>
+
+          <div data-testid={`${dataTestIdPrefix}-menu`}>
+            {options.map((option: { label: string; value: string }) => (
+              <div
+                key={option.value}
+                data-testid={option.value}
+                onClick={() => onSelect(option.value)}
+              >
```

Biome (2.3.13)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)